do/droplets.go

```diff
@@ -28,6 +28,8 @@ import (
 
 	"github.com/digitalocean/godo"
 	"github.com/digitalocean/godo/context"
+
+	"github.com/golang/glog"
```

Comment:
I think this project is intended to be as silent as can be, at least that's what it looks so far.
Since there is no `glog` usage at the `do` folder, this could probably be removed. Probably opening an issue to ask for non error logs management, and a different PR according to that PR is a good idea.